### PR TITLE
FIX #2823: Profile list activity label

### LINKED ISSUE #2823: [A11y] Add label for ProfileListActivity
## Task
Introduce label for `ProfileListActivity` as `Profile List`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by pravesh00**
## Explanation

Fixes #2823: Profile list activity label added

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.6.1'
+    classpath 'com.android.tools.build:gradle:4.1.3'
```
Comment (Victor-Titan) [inline build.gradle]: Gradle version should not be updated.

**Review by Victor-Titan — COMMENTED** [review]
Hey @pravesh00. There should not be a space in [ x ].

File: build.gradle (hunk 1 above)
Comment (pravesh00) [reply build.gradle]: ok sir I will make these changes

Comment (pravesh00) [reply build.gradle]: PTAL while reverting the gradle the distribution url was changed is it fine or i need to revert that too.
image

@Victor-Titan

Comment (Victor-Titan) [reply build.gradle]: @pravesh00 there should not be any changes in the gradle files.

Comment (pravesh00) [reply build.gradle]: can I make a new PR ?

**Pull request closed without merging**